src/Tribe/Views/V2/Customizer/Section/Global_Elements.php

```diff
@@ -0,0 +1,510 @@
+			'link_color'              => '#141827',
+			'background_color_choice' => 'transparent',
+			'background_color'        => '',
+			'accent_color'            => '#334aff',
+		];
+	}
+
+	/**
+	 * {@inheritdoc}
+	 */
+	public function setup_content_settings() {
+		return [
+			'font_family'               => [
+				'sanitize_callback'	   => 'sanitize_key',
+				'sanitize_js_callback' => 'sanitize_key',
+			],
+			'font_size'               => [
+				'sanitize_callback'	   => 'sanitize_key',
+				'sanitize_js_callback' => 'sanitize_key',
+			],
+			'font_size_base'          => [
+				'sanitize_callback'	   => 'sanitize_key',
+				'sanitize_js_callback' => 'sanitize_key',
+			],
+			'event_title_color'       => [
+				'sanitize_callback'	   => 'sanitize_hex_color',
+				'sanitize_js_callback' => 'maybe_hash_hex_color',
+			],
+			'event_date_time_color'   => [
+				'sanitize_callback'	   => 'sanitize_hex_color',
+				'sanitize_js_callback' => 'maybe_hash_hex_color',
+			],
+			'link_color'   => [
+				'sanitize_callback'	   => 'sanitize_hex_color',
+				'sanitize_js_callback' => 'maybe_hash_hex_color',
+			],
+			'background_color_choice' => [
+				'sanitize_callback'	   => 'sanitize_key',
+				'sanitize_js_callback' => 'sanitize_key',
+			],
+			'background_color'        => [
+				'sanitize_callback'	   => 'sanitize_hex_color',
+				'sanitize_js_callback' => 'maybe_hash_hex_color',
+			],
+			'accent_color'            => [
+				'sanitize_callback'	   => 'sanitize_hex_color',
+				'sanitize_js_callback' => 'maybe_hash_hex_color',
+			],
+		];
+	}
+
+	public function setup_content_headings() {
+		return [
+			'font_family_heading' => [
+				'priority'	 => 0,
+				'type'		 => 'heading',
+				'label'      => esc_html__( 'Select Font Family', 'the-events-calendar' ),
+			],
+			'font_size_heading' => [
+				'priority'	 => 5,
+				'type'		 => 'heading',
+				'label'      => esc_html__( 'Set Font Size', 'the-events-calendar' ),
+				'description' => esc_html_x(
+					'Choose a base font size. Event text will scale around the selected base size.',
+					'The description for the base font size setting.',
+					'the-events-calendar'
+				),
+			],
+			'font_color_heading' => [
+				'priority'	 => 10,
+				'type'		 => 'heading',
+				'label'      => esc_html__( 'Set Font Colors', 'the-events-calendar' ),
+			],
+			'global_elements_separator' => [
+				'priority'	 => 20,
+				'type'		 => 'separator',
+			],
+			'adjust_appearance_heading' => [
+				'priority'	 => 21,
+				'type'		 => 'heading',
+				'label'      => esc_html__( 'Adjust Appearance', 'the-events-calendar' ),
+			],
+		];
+	}
+
+	/**
+	 * {@inheritdoc}
+	 */
+	public function setup_content_controls() {
+		// Because Customizer doesn't show the default value.
+		if ( ! empty( $this->get_option( 'font_size_base' ) ) ) {
+			$font_size_base_value = $this->get_option( 'font_size_base' );
+		} else {
+			$font_size_base_value = $this->defaults[ 'font_size_base' ];
+		}
+
+		return [
```

Comment:
you have a mix of `_x()` and `esc_html_x()` for the choices in this array, should we align on one?